import pretty_errors

with open('./aerror.py', 'r') as file:
    print(file.read())